src/Network/PacketHandlers.cs

```diff
@@ -1870,8 +1921,12 @@ private static void ClientViewRange(Packet p)
         {
             World.ClientViewRange = p.ReadByte();
         }
-
-        private static void BulletinBoardData(Packet p)
+        public static string Truncate(string value, int maxLength)
```

Comment:
If this method is used 1 time, take the body and push it into the method caller.
Instead add it to Utility/StringHelper.cs class